docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -122,11 +144,15 @@ The configuration file for the Rackspace provider looks like:
      username: <rackspace username>
      key: <access key>
      keyname: <ssh-key-name>
-   metadata:
-     creator: <your-name>
 
 You will need a ssh agent running with access to the corresponding private key.
 
+Rackspace can use these dataset backends:
+
+  * :ref:`OpenStack<openstack-dataset-backend>`.
+  * :ref:`ZFS<zfs-dataset-backend>`.
+  * :ref:`Loopback<loopback-dataset-backend>`.
+
```

Comment:
Can you give examples of how a full config file for one of these would look? Otherwise I predict each person reading this for first time spends 20 minutes debugging obscure failures.